crates/bevy_ecs/src/storage/table.rs

```diff
@@ -893,6 +912,13 @@ impl Tables {
         }
     }
 
+    pub fn shrink_to_fit(&mut self) {
```

Comment:
This should probably be pub(crate). There's no way for a user of bevy_ecs to get a `&mut Tables` without being unsound.